source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2185,14 +2189,15 @@ export abstract class Action {
 
     id: string;
     title: string;
+    isPrimaryAction: boolean;
```

Comment:
Should be isPrimary, not isPrimaryAction

Reply:
This is outdated code. It is not from latest commit